modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -4561,6 +4561,62 @@ public void testUpdateObjectEntryWithAccountEntryRestricted2()
 						_simpleDTOConverterContext, objectDefinition,
 						objectEntry.getObjectEntryId()));
 			});
+
```

Comment:
@victorKammerer Please add a comment to clarify the test goal. Something like:

```
Users should be prevented from updating object entries restricted to 
accounts they do not belong to
```

Feel free to improve it.